src/flux/app-config.coffee

```diff
@@ -4,11 +4,16 @@ flux = require 'flux-react'
 
 {makeSimpleStore} = require './helpers'
 
+MinimizedPattern = /tutor\.min-\d+/
```

Comment:
Why not something like the following (and remove the `learning-guide/` prefix in `learning-guide-chart.coffee`?

```coffee
AppConfig =

  setAssetsPrefix: (prefix) ->
    @_assetsPrefix = prefix

  exports:
    getAssetsPrefix: ->
      return @_assetsPrefix if @_assetsPrefix?
      # http://caniuse.com/#feat=css-sel3 (substring selector is IE 9+)
      tutorStyle = document.head.querySelector('link[href*="tutor"]')
      # if we didn't find the styling css, use "/" string.
      # That'll cause the anchor test below to be a relative link and use the document's scheme/host/port
      href = if tutorStyle then tutorStyle.getAttribute('href') else "/"
      @_assetsPrefix = href[0.. href.lastIndexOf('/')]

    urlForResource: (path) ->
      prefix = this.exports.getAssetsPrefix.call(this)
      "#{prefix}/#{path}"
```